libraries/AP_GPS/AP_GPS_SBP2.cpp

```diff
@@ -288,6 +288,7 @@ AP_GPS_SBP2::_attempt_state_update()
         state.time_week_ms      = last_vel_ned.tow;
         state.hdop              = last_dops.hdop;
         state.vdop              = last_dops.vdop;
+        set_uart_timestamp(parser_state.msg_len);
```

Comment:
Why doesn't this one do the same as the Ublox?

Reply:
for SBP2 this is the only msg where we update both tow and state.last_gps_time_ms
I used the iTow for uBlox as we have a greater understanding of how that works.